python/trigger_ghebackup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+
+
+def get_pid(name: str) -> list:
+    """Returns a list of pids, can be empty.
+    @attention: inspired by http://stackoverflow.com/a/26688998 on 09/02/2017
+    @author: Padraic Cunningham"""
+    if name:
+        return list(map(int, subprocess.check_output(["pidof", name]).split()))
+    return []
+
+
+def start_ghe_backup(name: str) -> None:
+    """
+    Spawns process specified by given name
+    :param name: process to be spawned
+    """
+    try:
+        subprocess.Popen([name, '-v', '1>>', '/var/log/ghe-prod-backup.log', '2>&1'])
```

Comment:
Thanks for the input!

> You probably want to run this with shell = True and provide a string instead of a list, 
> otherwise the output will not work.

If the output works as intended remains unclear if tested with a test deployment.
That did not happen yet.

Based on 
* http://stackoverflow.com/a/3172488 
* http://stackoverflow.com/a/30831028

and similar resources on the web I decided _not_ to got for ```shell = True```.